src/target.zig

```diff
@@ -629,6 +629,14 @@ pub fn supportsFunctionAlignment(target: std.Target) bool {
     };
 }
 
+pub fn functionPointerMask(target: std.Target) ?u64 {
+    // 32-bit Arm uses the LSB to mean that the target function contains Thumb code.
+    return if (target.cpu.arch.isArm())
+        ~@as(u32, 1)
+    else
+        null;
+}
+
```

Comment:
MIPS can have an extension [MIPS16](https://en.wikipedia.org/wiki/MIPS_architecture#MIPS16), 
MIPS16e, MIPS16e2 or microMIPS which seem to be doing similar:
- [MicroChip docs](https://onlinedocs.microchip.com/oxy/GUID-471BA167-B55F-488B-A1CC-D88BAA7832CA-en-US-4/GUID-41B9C4DB-B9D8-44F0-BB45-BF514AB4BFCF.html)
- [gcc options](https://gcc.gnu.org/onlinedocs/gcc/MIPS-Options.html#index-mips16)

Reply:
Good find; I'll add MIPS to this function.